Bradm/154 (#3407)
FusionAuth/fusionauth-site: astro/src/content/blog/announcing-fusionauth-1-54.mdx

description: Version 1.54.0 of FusionAuth fixes an important SCIM issue, and brings free FusionAuth trials for everyone.
authors: Brad McCarty
image: /img/blogs/release-1-54/fusionauth-1-54.png
categories: Product
tags: SCIM, free trial, fusionauth cloud
excerpt_separator: "{/* more */}"
---

We recently announced the release of FusionAuth version 1.54.0 into the world. Our astute audience of developers will probably notice that this release looks a little different than most, so let's talk about it.

As is common in the software world, most minor releases include the addition of a notable, customer-facing feature:

- [1.53.0](/blog/announcing-fusionauth-1-53) added webhook logs and new updates for login lambdas. 
- [1.52.0](/blog/announcing-fusionauth-1-52) gave passkeys to the Community plan. 
- [1.51.0](/blog/announcing-fusionauth-1-51) was the release where we made major changes to theming.

1.54.0 takes a slight turn. There are still two significant updates to this release. Although we know that the individual audiences for each will be limited, they represent a big change to the FusionAuth software, so they still warrant a version update.

> Long-time users of FusionAuth will remember that we released SCIM support way back in [1.36.0](/blog/announcing-fusionauth-1-36). We've been pleased to see how many of you have put SCIM to work, but there was a nagging problem that we needed to address. The most recent release adds a change to the SCIM groups API that resolves a situation which could cause concurrency issues when making simultaneous calls to patch groups and group members. This revision makes sure that your SCIM group updates all show up when they should, and (critically) how they should.

Reviewer: ```suggestion
Long-time users of FusionAuth will remember that we released SCIM support way back in [1.36.0](/blog/announcing-fusionauth-1-36). We've been pleased to see how many of you have put SCIM to work, but there was a nagging problem that we needed to address. The most recent release adds a change to the SCIM groups API that resolves a situation which could cause concurrency issues when making simultaneous calls to patch groups and group members. This revision ensures SCIM group updates all show up when they should, and (critically) how they should.
```

> The next big change that we're announcing is that it's easier than ever to start using FusionAuth. In the past, there were certain criteria under which a new FusionAuth user could get a two-week trial of a paid version. Now anyone who signs up for FusionAuth Cloud gets a fully-featured, 30-day trial without even having to add in a credit card. We've been calling this Express Free Trial (EFT) internally, and so we've dubbed 1.54.0 the EFT Elephant.

Reviewer: ```suggestion
The next big change that we're announcing makes it easier than ever to start using FusionAuth. In the past, there were certain criteria under which a new FusionAuth user could get a two-week trial of a paid version. Now anyone who signs up for FusionAuth Cloud gets a fully-featured, 30-day trial without needing a credit card. We've been calling this Express Free Trial (EFT) internally, so we've dubbed 1.54.0 the EFT Elephant.
```
